### ISSUE TITLE
Add ability to make wrapping in context optional

### ISSUE THREAD

**Issue by iilyak** [jomaway/typst-linguify#7]
I am trying to use the linguify inside the calendar library I am working on. I would need to get string values for months and weekdays in different languages in typst code. Unfortunately it doesn't seem possible due to the fact that the linguify function creates context.

The simplest reproducer

```
#import "@preview/linguify:0.4.0": *

#let database = toml("lang.toml")

#let localize(weekday) = {
  linguify(weekday, from: database, default: weekday)
}

#set text(lang: "en")
#assert.eq("sunday", localize("sunday"))
```

this code fails with

```
equality assertion failed: value "sunday" was not equal to "context()"
```

**Comment by jomaway** [comment]
Can you try to use `_linguify()` instead. Not sure if this function is exported though. Then you need to provide context around the function calling it.

**Comment by iilyak** [comment]
The `_linguify` is private and not reachable.

**Comment by jomaway** [comment]
Ok I will fix this in the next version. But might take a while until I find some time.

**Comment by jomaway** [comment]
I published 0.4.1 if it is merged you can use 
```typst
#context {
  lflib._linguify(...)
}
```
which returns an dictionary with `{ "ok": "result" }`  or `{ "error" : "error_message" } `

you still need to provide context around the function call, because of checking for `text.lang`, 
maybe i will refactor it in the future to have an context free version if all parameters are passed in.
